### PR TITLE
Supporting devs section update

### PR DESCRIPTION
**Pull request by andrealbiac**
Replaced streams image with the new one in Supporting devs section.

Captura de pantalla 2024-09-26 a las 16 07 27

@austintgriffith is cooking some copy for this one

### CODE REVIEW

File: packages/nextjs/pages/index.tsx [hunk 1]
```diff
@@ -247,9 +247,9 @@ const Home: NextPage<{
               </div>
             </div>
           </div>
-          <div className="flex flex-col items-center">
-            <div className="max-w-md lg:max-w-none">
-              <Image src="/assets/support-high-impact-devs.png" alt="developers list" width={900} height={900} />
+          <div className="flex flex-col items-center lg:pl-16 xl:pl-12 2xl:pl-24 lg:-mr-12 xl:-mr-16 2xl:-mr-24">
+            <div className="max-w-md lg:max-w-md xl:max-w-xl 2xl:max-w-2xl">
+              <Image src="/assets/streams.png" alt="developers list" width={700} height={700} />
```
Comment (Pabl0cks) [inline packages/nextjs/pages/index.tsx:252]: Nitpick: you can update the `alt` tag to reflect the new image. We could either describe it literally (e.g. "Hourglass overlooking landscape") or describe the symbolism.

**Review by Pabl0cks — APPROVED** [review]
Looks good to me @andrealbiac! 👌

Deleted the old picture and added a nitpick comment 🙌

**Comment by andrealbiac** [comment]
Amazing! True forgot to delete the old one! And thanks for the tip, didn't know about the `alt` tag 🫶🏻

**Comment by andrealbiac** [comment]
Updated the alt tag with 'streams illustration' and fixed the text aligning in the Cohorts section, so that they match 🤝🏻

**Comment by Pabl0cks** [comment]
> Updated the alt tag with 'streams illustration' and fixed the text aligning in the Cohorts section, so that they match 🤝🏻

Nice catch! 👌

**Pull request merged**